.golangci.yml

```diff
@@ -0,0 +1,18 @@
+linters-settings:
+  misspell:
+    locale: US
+
+linters:
+    enable:
+    - golint
+    - goimports
+    - misspell
+    - gofmt
+    - unparam
+    - unconvert
+    - interfacer
+    disable:
+    - maligned
```

Comment:
aren't those disabled by default https://github.com/golangci/golangci-lint#disabled-by-default-linters--e--enable ?

Reply:
I found the following to not be enabled by default.
```
gofmt: Gofmt checks whether code was gofmt-ed. By default this tool runs with -s option to check for code 
goimports: Goimports does everything that gofmt does. Additionally it checks unused imports [fast: true, auto-fix: true]
interfacer: Linter that suggests narrower interface types [fast: false, auto-fix: false]
misspell: Finds commonly misspelled English words in comments [fast: true, auto-fix: true]
unconvert: Remove unnecessary type conversions [fast: true, auto-fix: false]
unparam: Reports unused function parameters [fast: false, auto-fix: false]
```

Tested with `$ golangci-lint linters` in dir with config and outside it.